packages/metro-react-native-babel-preset/src/configs/main.js

```diff
@@ -47,7 +47,7 @@ const getPreset = (src, options) => {
   const extraPlugins = [];
   if (!options.useTransformReactJSXExperimental) {
     extraPlugins.push([
-      require('@babel/plugin-transform-react-jsx', {useBuiltIns: true}),
```

Comment:
but why remove `useBuiltIns`?  [It's a standalone optimisation not related to either runtimes](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#usebuiltins) @danilobuerger